consensus/mint-client/src/config.rs

```diff
@@ -410,6 +431,17 @@ pub struct Config {
     pub command: Commands,
 }
 
+pub fn load_pub_key_from_pem(filename: &str) -> Result<Ed25519Public, String> {
```

Comment:
I think this can be merged into `load_key_from_pem` by making it generic, e.g.
`pub fn load_key_from_pem<K: DistinguishedEncoding>(filename: &str) -> Result<K, String>` and replacing `Ed25519Private::try_from_der` with `K::try_from_der`

Reply:
Fixed! Great sug :)